.github/workflows/create-release.yml

```diff
@@ -22,6 +22,10 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v3
 
+    # TODO: Should we test on multiple architectures?
```

Comment:
I think testing on both architectures makes sense here. The idea as I understand it is a smoke test to make sure the major parts are working, I feel like that means making sure both architectures are generally working OK.

It's probably less important with buildpackless builders since there are no buildpacks, but in general I think testing both archs at the builder level is a good place where we could detect potential problems.

Reply:
Yea this is fair

Reply:
done in the latest commit

Reply:
I only have the release notes generated on one of the architectures though, this might fail when it runs but we'll have to merge and see